modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro

```diff
@@ -382,6 +382,25 @@ definition {
 				echo("The string '${fileContentItem}' exists on this file");
 			}
 		}
+
+		FileUtil.delete("${directoryPath}/${fileName}");
+	}
+
+	@summary = "Verify that the state manager is present on MDF Request detail page"
+	macro viewFilterMDF {
+		Click(
+			key_filterOption = ${keyFilter},
+			locator1 = "AppBuilder#CHOOSE_SEARCH_FILTER_BY_OPTION");
+
+		Click(
+			keyPartnerOptions = ${keyPartner},
+			locator1 = "PRMMDFRequest#DROPDOWN_PARTNER");
```

Comment:
Hi @juliosilveira77, this variable shouldn't be 'keyPartner', since this macro can be used to another filter types. this variable should be 'filterType'

Reply:
Also, change the path name to FILTER_TYPE